README.md

```diff
@@ -1,13 +1,36 @@
 # Express API Project
 
-Replace this readme with your own information about your project. 
+For this week I created an API with my dataset collected of Kaggle.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This data has all the best artworks of all the time with images per artist. 
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+The database that I chose has around 7.000 images and the problem for this project was how to show that image. To solve that I used standard methods of express to set the path of the images. 
+
```

Comment:
![image](https://user-images.githubusercontent.com/50926159/119458506-1d782580-bd3d-11eb-89f0-0a9a491bd1d6.png)
 Github is not a place to store images for a big app. You should use an external picture hosting provider and just populate your app with URL strings. In the future, you will build a server, and then you will send your data straight to the server storage/database no into your code. 
 Now I can't even comment on your code because it will not be shown to me on this listing. I will look at it on your GitHub account, but there I can't comment on it.